File: js/task1.js
const sumDivisibleOf3and5 = () => {
	let sum = 0;

	for (let i = 0; i < 1000; i++) {
		if (!(i % 3) || !(i % 5)) {
			sum += i;
		}
	}

	return sum;
}

console.log(sumDivisibleOf3and5());

File: python/task1.py
def divis3(number):
    while(number & (~3)):
        sum = 0
        while(number):
            sum += number & 3
            number >>= 2
        number = sum
    return (number == 0 or number == 3)


def divis5(number):
    divisor = 0x33333333
    return ((1 + ((divisor * number) >> 32)) * 5) == number

def getSum():
    return sum(i for i in range(1000) if (divis3(i) or divis5(i)))

if __name__ == "__main__":
    print(getSum())
